script/output/holesky/Deploy_RewardsCoordinator.holesky.config.json

```diff
@@ -3,20 +3,17 @@
     "avsDirectory": "0x055733000064333CaDDbC92763c58BF0192fFeBf",
     "avsDirectoryImplementation": "0xEF5BA995Bc7722fd1e163edF8Dc09375de3d3e3a",
     "baseStrategyImplementation": "0xFb83e1D133D0157775eC4F19Ff81478Df1103305",
-    "beaconOracle": "0x4C116BB629bff7A8373c2378bBd919f8349B8f25",
-    "delayedWithdrawalRouter": "0x642c646053eaf2254f088e9019ACD73d9AE0FA32",
-    "delayedWithdrawalRouterImplementation": "0xcE8b8D99773a718423F8040a6e52c06a4ce63407",
     "delegationManager": "0xA44151489861Fe9e3055d95adC98FbD462B948e7",
     "delegationManagerImplementation": "0x83f8F8f0BB125F7870F6bfCf76853f874C330D76",
     "eigenLayerPauserReg": "0x85Ef7299F8311B25642679edBF02B62FA2212F06",
     "eigenLayerProxyAdmin": "0xDB023566064246399b4AE851197a97729C93A6cf",
     "eigenPodBeacon": "0x7261C2bd75a7ACE1762f6d7FAe8F63215581832D",
-    "eigenPodImplementation": "0xe98f9298344527608A1BCC23907B8145F9Cb641c",
+    "eigenPodImplementation": "0x10ad7e30e3F52076C8462D573530f4461377319c",
```

Comment:
why are all of these updating?

Reply:
its a result of using the `eigenlayer_addresses.config.json` which was updated from PEPE deploy scripts. Meaning this output file is getting updated now.

Reply:
why didn't merging pepe deploy into dev do this?

Reply:
because pepe deploy didn't touch the rewards coordinator file (why would it)?

"what config file" we need to edit when deploying is very unclear :sweat: